changelogs/unreleased/3192-pydantic-anyurl-doc-rendering.yml

```
@@ -0,0 +1,5 @@
+---
+description: Fixed documentation rendering (OpenAPI) bug where AnyUrl types are shown as object instead of string.
+issue-nr: 3192
+change-type: patch
+destination-branches: [master, iso4]
```

```suggestion
destination-branches: [iso4]
```
If I understand correctly this has already been merged into master.

---

Indeed, I didn't thought of removing that